### PR TITLE
Fixed typos in RIT professors. Added new RIT tenure-track professors.…

### PR DESCRIPTION
**Pull request by zzm7000**
… Their qualifications can be verified at http://phd.gccis.rit.edu/people.html

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -628,6 +629,7 @@ Amiya Nayak,University of Ottawa,http://www.site.uottawa.ca/~anayak,ttUIYysAAAAJ
 Amiya R. Nayak,University of Ottawa,http://www.site.uottawa.ca/~anayak,ttUIYysAAAAJ
 Amjad Hudaib,University of Jordan,http://eacademic.ju.edu.jo/ahudaib,WezwTlcAAAAJ
 Amjed Tahir,Massey University,http://www.massey.ac.nz/massey/expertise/profile.cfm?stref=481450,bEw-I00AAAAJ
+Amlan Ganguly,Rochester Institute of Technology,https://people.rit.edu/axgeec/,tuWDm5kAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:632]: Not in CS.

File: csrankings.csv [hunk 2]
```diff
@@ -941,7 +943,7 @@ Anne Bouillard,Ecole Normale Superieure,http://www.di.ens.fr/~bouillar,NOSCHOLAR
 Anne Brüggemann-Klein,TU Munich,http://www.professoren.tum.de/en/brueggemann-klein-anne,NOSCHOLARPAGE
 Anne Condon,University of British Columbia,http://www.cs.ubc.ca/~condon,SO_j4zwAAAAJ
 Anne Driemel,TU Eindhoven,http://www.win.tue.nl/~adriemel,JhIotuYAAAAJ
-Anne Haake,Rochester Institute of Technology,https://www.rit.edu/gccis/anne-haake,d0JcWVIAAAAJ
+Anne R. Haake,Rochester Institute of Technology,https://www.rit.edu/gccis/anne-haake,d0JcWVIAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:946]: Not in CS. https://www.cs.rit.edu/people#faculty

File: csrankings.csv [hunk 3]
```diff
@@ -4524,6 +4526,7 @@ Hanan Lutfiyya,Western University,http://www.csd.uwo.ca/faculty/hanan,NOSCHOLARP
 Hanan Samet,University of Maryland - College Park,https://www.cs.umd.edu/people/hjs,ATOfqHEAAAAJ
 Hanghang Tong,Arizona State University,https://cidse.engineering.asu.edu/directory/tong-hanghang,RaINcuUAAAAJ
 Hanjun Kim,POSTECH,http://corelab.postech.ac.kr/~hanjun,rdQ5AokAAAAJ
+Hanif Rahbari,Rochester Institute of Technology,https://www.rit.edu/gccis/computingsecurity/people/hanif-rahbari,0lhBjI4AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:4529]: Not in CS. https://www.cs.rit.edu/people#faculty

File: csrankings.csv [hunk 4]
```diff
@@ -6808,6 +6812,7 @@ Kristian M. Müller,Bielefeld University,https://ekvv.uni-bielefeld.de/pers_publ
 Kristin J. Dana,Rutgers University,https://www.cs.rutgers.edu/people/faculty/affiliated-faculty,FLmgLo0AAAAJ
 Kristof Van Laerhoven,University of Freiburg,https://es.informatik.uni-freiburg.de/team/kristof-van-laerhoven,iPI39lEAAAAJ
 Kristofer S. J. Pister,University of California - Berkeley,https://www2.eecs.berkeley.edu/Faculty/Homepages/pister.html,NOSCHOLARPAGE
+Kristen Shinohara,Rochester Institute of Technology,http://www.kristenshinohara.com/,phGrIKYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:6815]: Not in CS. https://www.cs.rit.edu/people#faculty

File: csrankings.csv [hunk 5]
```diff
@@ -8747,6 +8753,7 @@ Nate Foster,Cornell University,http://www.cs.cornell.edu/~jnfoster,AH5j40kAAAAJ
 Nathalie Revol,Ecole Normale Superieure de Lyon,http://perso.ens-lyon.fr/nathalie.revol,f00lNLUAAAAJ
 Nathaly Verwaal,University of Calgary,http://contacts.ucalgary.ca/info/cpsc/profiles/102-3419,NOSCHOLARPAGE
 Nathan Beckmann,Carnegie Mellon University,http://www.cs.cmu.edu/~beckmann,zkdjit0AAAAJ
+Nathan D. Cahill,Rochester Institute of Technology,https://people.rit.edu/ndcsma/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:8756]: Not in CS. https://www.cs.rit.edu/people#faculty

**Review by emeryberger — CHANGES_REQUESTED** [review]
There are numerous people who are not in CS. I stopped counting. They need to be removed because they do not meet the inclusion criteria (or you need to provide evidence that they do).

**Comment by zzm7000** [comment]
At RIT, there is only one CS PhD program called PhD in Computing and Information Science. This program is at the College level not at computer science department level. Faculty members in Golisano College of Computing and Information Science belong to different department including computer science, computing security, etc. http://phd.gccis.rit.edu lists all the faculty members who can solely advise PhD students in Computing and Information science.

Hope it clarifies. Let me know what you think. Thank you!

**Comment by zzm7000** [comment]
For example, my tenure home is in the computing security department. But, the PhD degrees are the same for all the student in the college.

**Comment by emeryberger** [comment]
Please split this into two PRs: one to correct existing entries / add faculty members in the CS department, and another to add these other non-CS dept. faculty members. I can straightforwardly merge the former; the latter may take some investigation. Thanks.

**Comment by fycus-tree** [comment]
The link for people who can advise PhD students in Computing & Information Sciences is here http://phd.gccis.rit.edu/people.html
and includes people from every College of Computing sub-department. As a PhD applicant, they'd all be effectively one department.

**Comment by emeryberger** [comment]
Thanks, @fycus-tree. All clear.

**Review by emeryberger — APPROVED** [review]

**Pull request merged**